docs/architecture/incremental.md

```diff
@@ -2,15 +2,15 @@
 
 ## Overview
 
-Incremental - Append syncs in Airbyte allow sources to replicate only _new_ or _modified_ data. This prevents re-fetching data that you have already replicated from a source. We will call this set of new or updated records the delta going forward.
+Incremental - Append syncs in Airbyte allow sources to replicate only _new_ or _modified_ data. This prevents re-fetching data that you have already replicated from a source. Going forward, we will call this set of new or updated records the delta.
 
-In this flavor of incremental records in the warehouse will never be deleted or mutated. A new copy of any new or updated records is appended to the data in the warehouse. This means you can find multiple copies of the same record twice in the warehouse and will need to de-duplicate them yourself. We provided an "at least once" guarantee of replicating each record that is present when the sync runs.
+In this flavor of incremental, records in the warehouse will never be deleted or mutated. A copy of each new or updated records is appended to the data in the warehouse. This means you can find multiple copies of the same record twice in the warehouse and will need to de-duplicate them yourself. We provided an "at least once" guarantee of replicating each record that is present when the sync runs.
```

Comment:
in DBT, we could deduplicate rows in the future

Or point to the user how to do it in the DBT models we generate...

Reply:
Yeah we should write this tutorial! @cgardens do we already have an issue for that?

Reply:
no we don't have one yet. please go ahead and make it!